aspnetcore/security/authentication/claims.md

```diff
@@ -21,9 +21,9 @@ This article covers the following areas:
 * Reset the claims namespaces
 * Customize, extend the claims using <xref:Microsoft.AspNetCore.Authentication.IClaimsTransformation.TransformAsync%2A>
 
-## Mapping claims using Open ID Connect authentication
+## Mapping claims using OpenID Connect authentication
 
-The profile claims can be returned in the `id_token` which is returned after a successful authentication. The ASP.NET Core client app just needs to request the profile scope. When using the `id_token` for claims, no extra claims mapping is required.
+The profile claims can be returned in the `id_token` which is returned after a successful authentication. The ASP.NET Core client application just needs to request the profile scope. When using the `id_token` for claims, no extra claims mapping is required.
```

Comment:
More nits here. Try ...... (with "app" ... that's ok) ...

```suggestion
The profile claims can be returned in the `id_token`, which is returned after a successful authentication. The ASP.NET Core client app only requires the profile scope. When using the `id_token` for claims, no extra claims mapping is required.
```

Reply:
thanks for the review, updated with you improvements